cluster-scope/bundles/jaeger-operator/kustomization.yaml

```diff
@@ -3,5 +3,7 @@ kind: Kustomization
 namespace: openshift-distributed-tracing
 resources:
   - ../../base/core/namespaces/openshift-distributed-tracing
-  - ../../base/operators.coreos.com/subscriptions/jaeger-operator
-  - ../../base/operators.coreos.com/operatorgroups/jaeger-operator
+  - ../../base/operators.coreos.com/subscriptions/openshift-distributed-tracing-platform
+  - ../../base/operators.coreos.com/operatorgroups/openshift-distributed-tracing-platform
```

Comment:
destination does not exist, should be: 

`../../base/operators.coreos.com/operatorgroups/openshift-distributed-tracing`

Reply:
fixed, I've set all jaeger-operator files back to original, I'm only adding opentelemetry-collector-operator files in this PR